openassessment/templates/openassessmentblock/grade/oa_assessment_feedback.html

```diff
@@ -0,0 +1,41 @@
+{% load i18n %}
+{% spaceless %}
+    {% if assessment %}
+        <li class="answer feedback">
+            {% if assessment.individual_assessments %}
+                {% for individual_assessment in assessment.individual_assessments %}
+                    {% if individual_assessment.feedback %}
```

Comment:
I don't understand this section of the template (lines 5-26). When is this shown?